src/containers/VDiskPage/VDiskPage.tsx

```diff
@@ -218,19 +242,29 @@ export function VDiskPage() {
     };
 
     const renderTabs = () => {
-        const vDiskParamsDefined =
-            valueIsDefined(nodeId) && valueIsDefined(pDiskId) && valueIsDefined(vDiskSlotId);
+        const vDiskExtendedParamsDefined = !isNil(nodeId) && !isNil(pDiskId) && !isNil(vDiskSlotId);
+
+        const vDiskBasicParamsDefined = !isNil(vDiskId);
```

Comment:
Looks like you may leave only `!isNil(vDiskId)`, I don't see why you the rest params are necessary